src/Exception/YotiClientException.php

```diff
@@ -4,6 +4,9 @@
 
 namespace Yoti\Exception;
 
+/**
+ * @deprecated no longer in use
```

Comment:
The Yoti Client no longer throws this exception as validation was moved to in the last major release